hr_public_holidays/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,8 @@
+<odoo>
+
+  <record id="public_holiday_color" model="ir.config_parameter">
+    <field name="key">calendar.public_holidays_color</field>
+    <field name="value">#cc9bef</field>
```

Comment:
Can you change the default color to a 'Odoo' color? This way it feels by default more natural.

Reply:
what exactly do you call an 'Odoo' color? :slightly_smiling_face:

Reply:
the purple one maybe?

Reply:
isn't the original one purple enough already? :smile: 
![image](https://user-images.githubusercontent.com/34241889/45617575-6dc26400-ba7b-11e8-9a1a-b4df786792a7.png)